### PR TITLE
quick clean up

### PR DESCRIPTION
**Pull request by wzrdtales**
As talked the transfer from dgonzalez/visigoth-old#2

not sure where it should be opened up on, its there in both places now:
https://github.com/rjrodger/visigoth/pull/3

### CODE REVIEW

File: examples/econ-refused/package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "visigoth-econ-refused-example",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "main": "econ-refused.js",
+  "author": "",
+  "license": "ISC",
```
Comment (dgonzalez) [inline examples/econ-refused/package.json]: License should be MIT (as per base repo)

**Review by dgonzalez — CHANGES_REQUESTED** [review]
Looks good. Just change the license and I will merge it. This is the correct repo.

File: examples/econ-refused/package.json (hunk 1 above)
Comment (dgonzalez) [reply examples/econ-refused/package.json]: Applies for all the other examples.

Comment (wzrdtales) [reply examples/econ-refused/package.json]: Sure will Update this just quickly Cherry picked the old One

**Comment by wzrdtales** [comment]
@dgonzalez Pushed the requested changes out. About the repo constellation. It is quite a struggle to open a PR in this constellation as you always end up by default in rjdoger ones. Is there a special reason for this structure?

**Comment by dgonzalez** [comment]
No, repo will be transferred to my account. This is the right one. Thanks for your changes!

**Pull request merged**